src/components/MakeWidget/subcomponents/RateField/RateField.styles.tsx

```diff
@@ -0,0 +1,37 @@
+import styled, { css } from "styled-components/macro";
+
+import { fontMono } from "../../../../style/themes";
+
+export const RateFieldStyle = css`
+  color: ${({ theme }) => theme.colors.lightGrey};
+  text-transform: uppercase;
+  font-size: 0.75rem;
+`;
+
+export const Button = styled.div`
+  cursor: pointer;
+`;
+
+export const Text = styled.div`
+  /* text will not vertically align according to figma spec*/
+  margin-top: -2px;
+`;
+
+export const Wrapper = styled.div`
+  ${RateFieldStyle}
+  display: flex;
+  align-items: center;
+  gap: 0.375rem;
+  margin-left: auto;
+  margin-top: 1rem;
```

Comment:
```suggestion
```

The parent should style the margin. That's why you should put className in props.